api4/team_test.go

```diff
@@ -1452,6 +1452,24 @@ func TestInviteUsersToTeam(t *testing.T) {
 			}
 		}
 	}
+
+	cfg := app.GetConfig()
+	cfg.TeamSettings.RestrictCreationToDomains = "@example.com"
+	cfg, _ = th.SystemAdminClient.UpdateConfig(cfg)
```

Comment:
This is my first look at Mattermost's code and Go-code in general, I'll change this as well.